openresty-build-tools/kong-ngx-build

```diff
@@ -275,6 +275,13 @@ main() {
           notice "Downloaded: $(sha256sum "openresty-$OPENRESTY_VER.tar.gz")"
         fi
         tar -xzf openresty-$OPENRESTY_VER.tar.gz
+        
+        # change the luajit version text
+        pushd openresty-$OPENRESTY_VER/bundle
+        lj_dir=$(ls -d LuaJIT*)
+        lj_update_time=$(echo ${lj_dir} | sed -e 's/LuaJIT-[[:digit:]]\+.[[:digit:]]\+-\([[:digit:]]\+\)/\1/')
+        sed -i "s/#define[[:space:]]\+LUAJIT_VERSION[[:space:]]\+\"LuaJIT \([[:digit:]]\+.[[:digit:]]\+\).[[:digit:]]\+.\+\"/#define LUAJIT_VERSION		\"LuaJIT v\1-${lj_update_time}\"/" ${lj_dir}/src/luajit.h
+        popd
```

Comment:
nit/style: lines between `pushd`/`popd` should be indented 2 spaces to match the style of the rest of this script

```suggestion
        pushd openresty-$OPENRESTY_VER/bundle
          lj_dir=$(ls -d LuaJIT*)
          lj_update_time=$(echo ${lj_dir} | sed -e 's/LuaJIT-[[:digit:]]\+.[[:digit:]]\+-\([[:digit:]]\+\)/\1/')
          sed -i "s/#define[[:space:]]\+LUAJIT_VERSION[[:space:]]\+\"LuaJIT \([[:digit:]]\+.[[:digit:]]\+\).[[:digit:]]\+.\+\"/#define LUAJIT_VERSION		\"LuaJIT v\1-${lj_update_time}\"/" ${lj_dir}/src/luajit.h
        popd
```

Reply:
Some other bash/style notes here (feel free to disregard though)...

---

```bash
lj_update_time=$(echo ${lj_dir} | sed -e 's/LuaJIT-[[:digit:]]\+.[[:digit:]]\+-\([[:digit:]]\+\)/\1/')
```

This can be expressed more succinctly (and without  by using bash string functions:

```bash
lj_update_time=${lj_dir##*-}
```

---

```bash
sed -i "s/#define[[:space:]]\+LUAJIT_VERSION[[:space:]]\+\"LuaJIT \([[:digit:]]\+.[[:digit:]]\+\).[[:digit:]]\+.\+\"/#define LUAJIT_VERSION		\"LuaJIT v\1-${lj_update_time}\"/" ${lj_dir}/src/luajit.h
```

Maybe consider breaking this onto multiple lines and using the extended regex support in sed to make this pattern more readable with less escaping:

> -E, -r, --regexp-extended
>
>    use extended regular expressions in the script (for portability use POSIX -E).

(note: I didn't test this)

```bash
sed -i -r \
  -e "s/#define +LUAJIT_VERSION +\"LuaJIT ([0-9]+[.][0-9]+)[.][0-9]+.+\"/#define LUAJIT_VERSION		\"LuaJIT v\1-${lj_update_time}\"/" \
  ${lj_dir}/src/luajit.h
```